db-service/lib/cqn2sql.js

```diff
@@ -547,12 +548,12 @@ class CQN2SQLRenderer {
     if (data)
       for (let c in data)
         if (!elements || (c in elements && !elements[c].virtual)) {
-          columns.push({ name: c, sql: this.val({ val: data[c] }) })
+          columns.push({ name: c, sql: this.val({ val: data[c] }) + '' })
```

Comment:
Yes → likely just that:
```js
      case 'boolean':
        return String(val)
      case 'number':
        return String(val) // REVISIT for HANA
```

Reply:
Btw: 
```js
        if (val instanceof require('stream').Readable) {
```

→ please add a static const import for `Readable`, instead of relying on `require.cache` in such a high-frequently used function